source/draw/RasterBgr.ooc

```diff
@@ -63,13 +63,13 @@ RasterBgr: class extends RasterPacked {
 	}
 	distance: func (other: Image) -> Float {
 		result := 0.0f
-		if (!other)
+		if (!other || (this size != other size))
 			result = Float maximumValue
-//		else if (!other instanceOf?(This))
-//			FIXME
-//		else if (this size != other size)
-//			FIXME
-		else {
+		else if (!other instanceOf?(This)) {
```

Comment:
Unfortunately no, as `RasterImage` does not have a generic method to access pixel value at (x, y).